src/CoreMedia/CMSync.cs

```diff
@@ -477,6 +477,7 @@ internal CMClockOrTimebase ()
 		public CMClockOrTimebase (IntPtr handle)
 		{
```

Comment:
This works, but the usual pattern is to call the other ctor:

    : this (handle, false)